cmd/kfulint/testdata/builtin-shadow/checker_tests.go

```diff
@@ -1,63 +1,163 @@
 package checker_test
 
-func assigningToBuiltinFunctions() {
-	/// assigning to builtin function: append
+func assigningToBuiltinIdentifiers() {
+	// Types:
```

Comment:
By the way, you can split different types of checks (types, constants, funcs, etc.) to different functions.